src/app/deposits/fixed-deposits/fixed-deposit-account-view/fixed-deposit-account-view.component.html

```diff
@@ -90,10 +90,12 @@ <h3> {{"labels.heading.Account Overview" | translate}} </h3>
       [active]="IRC.isActive">
       {{"labels.inputs.Interest Rate Chart" | translate }}
```

Comment:
If this is a loan product view enhancement, please do not modify the fixed deposit account view... Raise a new PR for these changes!